sdk/src/main/java/software/amazon/awssdk/iot/discovery/DiscoveryClientConfig.java

```diff
@@ -27,6 +28,24 @@ public DiscoveryClientConfig(
         this.region = region;
         this.maxConnections = maxConnections;
         this.proxyOptions = proxyOptions;
+        this.ggServerName = "";
+    }
+
+    public DiscoveryClientConfig(
+            final ClientBootstrap bootstrap,
+            final TlsContextOptions tlsContextOptions,
+            final SocketOptions socketOptions,
+            final String region,
```

Comment:
don't bother having `region` in this constructor, it's unused if we're setting ggServerNam